.slather.yml

```diff
@@ -0,0 +1,10 @@
+coverage_service: cobertura_xml
+xcodeproj: platforms/ios/example/Wysiwyg.xcodeproj
+scheme: Wysiwyg
+source_directory: platforms/ios/example
+output_directory: platforms/ios/example
+build_directory: platforms/ios/example/DerivedData
+ignore:
+  - Wysiwyg/Mocks/*
+  - Wysiwyg/AppDelegate.swift
```

Comment:
I guess these two should be `example/Wysiwyg/...` but should we just exclude the entire `example` directory from coverage stats ? Supposing that all we care about is the `lib` folder which is what is actually extracted to our Swift Package repo.

Reply:
I think from he examples I saw the ignores were relative to the source_directory above, but will double check this.

But yea you are probably right we probably don't need to do coverage on the sample, as it's probably more valuable to be testing the libraries integration in the real work examples (EI and EIX).